torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -393,7 +393,7 @@ Tensor prod_backward(const Tensor& grad, const Tensor& input, const Tensor& resu
   }
   Tensor zero_idx = (input == 0).nonzero();
   if (zero_idx.numel() == 0) {
-    return (grad * result) / input;
+    return grad * (result / input).conj();
```

Comment:
What is the precision implication of flipping this order? I have https://github.com/pytorch/pytorch/issues/43414 in mind

Reply:
I am not sure they are comparable. The reason being one order is better then `input[i]` is greater than 1, the other when `input[i]` is less than 1. And it is only entry-wise dependent...

Reply:
The different in the other one was that we were squaring the denominator before doing the division. BUt since there is no squaring it should be fine here right?

Reply:
But if you feel more comfortable with the original order, I could make it like
`grad * result.conj() / input.conj()`

Reply:
I think the new one will work. It just that it reminded me of that issue and I didn't want to introduce any regression.

Reply:
I think you are right. I will restore the previous order so that the change does not become bc-breaking.